app/src/main/java/org/fossasia/openevent/app/qrscan/ScanQRPresenter.java

```diff
@@ -50,17 +48,31 @@ private void processBarcode(String barcode) {
         scanQRView.showBarcodeData(barcode);
 
         Observable.fromIterable(attendees)
+            .filter(attendee -> attendee.getOrder() != null)
             .filter(attendee -> (attendee.getOrder().getIdentifier() + "-" + attendee.getId()).equals(barcode))
             .subscribeOn(Schedulers.computation())
             .observeOn(AndroidSchedulers.mainThread())
-            .subscribe(attendee -> scanQRView.onScannedAttendee(attendee));
+            .subscribe(attendee -> {
+                if(scanQRView == null)
+                    return;
+
+                scanQRView.onScannedAttendee(attendee);
+            });
+    }
+
+    public void setAttendees(List<Attendee> attendees) {
+        this.attendees = attendees;
     }
 
     @Override
     public void attach() {
```

Comment:
Yeah it's not related to this PR.
Actually I found this while reading `shouldNotAccessViewAfterDetach()`. where presenter is detached first and then attached again. Then view's accessibility is tested. 

I will open new issue for this. :+1: